sdks/typescript/packages/sdk/package.json

```
@@ -42,10 +59,17 @@
     "node": ">=18.0.0"
   },
   "peerDependencies": {
+    "react": "^18.0.0 || ^19.0.0-0 || ^19.0.0",
     "undici": "^6.19.2"
   },
+  "peerDependenciesMeta": {
+    "react": {
+      "optional": true
+    }
+  },
   "devDependencies": {
     "@clockworklabs/test-app": "workspace:^",
+    "@types/react": "^19.1.13",
```

```suggestion
    "@types/react": ">=18.0.0",
```
We should use types from the lowest supported version